bundles/org.openhab.binding.modbus.e3dc/README.md

```diff
@@ -0,0 +1,367 @@
+Number    E3DC_String1A                 "E3DC String 1 Ampere"  (e3dc)  { channel="modbus:e3dc:device:powerplant:strings#string1-dc-current" }
+Number    E3DC_String2A                 "E3DC String 2 Ampere"  (e3dc)  { channel="modbus:e3dc:device:powerplant:strings#string2-dc-current" }
+Number    E3DC_String3A                 "E3DC String 3 Ampere"  (e3dc)  { channel="modbus:e3dc:device:powerplant:strings#string3-dc-current" }
+Number    E3DC_String1W                 "E3DC String 1 Watt"    (e3dc,persist)  { channel="modbus:e3dc:device:powerplant:strings#string1-dc-output" }
+Number    E3DC_String2W                 "E3DC String 2 Watt"    (e3dc,persist)  { channel="modbus:e3dc:device:powerplant:strings#string2-dc-output" }
+Number    E3DC_String3W                 "E3DC String 3 Watt"    (e3dc,persist)  { channel="modbus:e3dc:device:powerplant:strings#string3-dc-output" }
+
+String    E3DC_EMS_Status                       "E3DC EMS Status"                      (e3dc)  { channel="modbus:e3dc:device:powerplant:emergency#emergency-power-status" }
+Switch    E3DC_EMS_BatteryChargingLock          "E3DC EMS Battery Charging Locked"     (e3dc)  { channel="modbus:e3dc:device:powerplant:emergency#battery-charging-lock" }
+Switch    E3DC_EMS_BatteryDischargingLock       "E3DC EMS Battery Discharging Locked"  (e3dc)  { channel="modbus:e3dc:device:powerplant:emergency#battery-discharging-lock" }
+Switch    E3DC_EMS_EmergencyPowerPossible       "E3DC EMS Emergency Power possible"    (e3dc)  { channel="modbus:e3dc:device:powerplant:emergency#emergency-power-possible" }
+Switch    E3DC_EMS_WeatherPredictedCharging     "E3DC EMS Weather Predicted Charging"  (e3dc)  { channel="modbus:e3dc:device:powerplant:emergency#weather-predicted-charging" }
+Switch    E3DC_EMS_RegulationStatus             "E3DC EMS Regulation Status"           (e3dc)  { channel="modbus:e3dc:device:powerplant:emergency#regulation-status" }
+Switch    E3DC_EMS_ChargeLockTime               "E3DC EMS Charge Lock Time"            (e3dc)  { channel="modbus:e3dc:device:powerplant:emergency#charge-lock-time" }
+Switch    E3DC_EMS_DischargeLockTime            "E3DC EMS Discharge Lock TIme"         (e3dc)  { channel="modbus:e3dc:device:powerplant:emergency#discharge-lock-time" }
+```
+
+### Sitemap
+
+```
+sitemap E3DC label="E3DC Binding Sitemap" {
+  Frame label="Info" {
+    Text    item=E3DC_ModbusId                  label="Modbus-ID [%s]"
+    Text    item=E3DC_ModbusFirmware            label="Modbus Firmware [%s]"
+    Text    item=E3DC_SupportedRegisters        label="Registers [%s]"
+    Text    item=E3DC_Manufacturer              label="Manufacturer [%s]"
+    Text    item=E3DC_ModelName                 label="Model Name [%s]"
+    Text    item=E3DC_Firmware                  label="Firmware [%s]"
+    Text    item=E3DC_SerialNumber              label="Serial Number[%s]"
+  }
+  
+    Frame label="Power Producer" {
+      Text    item=E3DC_PVPower                 label="PV Power[%s]"
```

Comment:
Did you test this? I think you have to specify the unit.
```suggestion
      Text    item=E3DC_PVPower                 label="PV Power [%d %unit%]"
```

Reply:
No, I didn't tested this. I'll apply the changes and check the results.

Reply:
New sitemap looks good. Units are displayed and values are adapted to their real values. Power and Potential as integer, current as float